go.mod

```diff
@@ -3,6 +3,6 @@ module github.com/aquasecurity/trivy-plugin-aqua
 go 1.18
```

Comment:
### :warning: Aqua detected vulnerability in your code  
**Vulnerability ID:** CVE-2022-2582   
**Check Name:** The AWS S3 Crypto SDK sends an unencrypted hash of the plaintext along ...   
**Severity:** MEDIUM   
**Fixed Version:** 1.34.0   
**Description:** The AWS S3 Crypto SDK sends an unencrypted hash of the plaintext alongside the ciphertext as a metadata field. This hash can be used to brute force the plaintext, if the hash is readable to the attacker. AWS now blocks this metadata field, but older SDK versions still send it.  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/nvd/cve-2022-2582